content/Software and Downloads/Compilation/If-your-sketch-doesnt-compile.md

```diff
@@ -0,0 +1,242 @@
+---
+title: "If your sketch doesn't compile"
+---
+
+Learn about compilation errors in your sketch and how to resolve them.
+
+---
+
+## Quick checks
+
+* Make sure your error occurs during compilation by clicking ![Verify button](img/symbol_verify.png) **Verify** instead of ![Upload button](img/symbol_upload.png) **Upload**. This will compile the sketch without attempting to upload it. If your error only occurs when uploading, see [Errors when uploading a sketch](https://support.arduino.cc/hc/en-us/articles/4403365313810-Errors-when-uploading-a-sketch).
+
+* A successful compilation will always finish with this message:
```

Comment:
```suggestion
* A successful compilation will always finish with this message (the storage space and memory values might differ depending on the board used):
```
Just to make sure that the user don't assume that the message has to be _exactly_ the same for successful compilation